File: web3rpc/rpc-specs/code-samples/python/governance/getStakingInfo.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

blockNumber = "latest"

w3 = Web3(Web3.HTTPProvider(host))
governance_response = w3.governance.get_staking_info(blockNumber)

print(governance_response)

File: web3rpc/rpc-specs/code-samples/javascript/governance/getStakingInfo.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const blockNumber = "latest";

  provider.governance
    .getStakingInfo(blockNumber, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
